lib/experimental/fonts/font-library/font-library.php

```diff
@@ -56,10 +56,18 @@ function wp_register_font_collection( $config ) {
 	}
 }
 
-// @core-merge: This code needs to be removed.
 add_action(
 	'enqueue_block_editor_assets',
 	function () {
 		wp_add_inline_script( 'wp-block-editor', 'window.__experimentalFontLibrary = true', 'before' );
 	}
 );
+
+$default_font_collection = array(
```

Comment:
Since there is now a default font collection, I think [this test](https://github.com/WordPress/gutenberg/blob/f3a6ff10f1c2055fdf4f62d2e851778d1a0f6db2/phpunit/tests/fonts/font-library/wpFontLibrary/getFontCollections.php#L38) needs to be updated — specifically the count of collections to expect after registering additional collections.

Reply:
That test tests the capabilities to register font collections and counts the 2 mock collections registered in the test. It is not testing the inclusion of the Google fonts collections.

Reply:
You were right as we are using a static class there is only one list of font collections. I refactored the tests to work as expected.